app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -19,61 +19,103 @@ inset_text:
+Getting a PGCE can help you:
 
-Fees are usually around £9,250 but you can:
+* get a better understanding of teaching practice and theory
+* earn up to 60 credits towards a master’s - you may want to do a master’s in education, for example, for your professional development
+* teach in other countries
 
-- get [funding that you do not have to pay back](/funding-and-support/scholarships-and-bursaries) for certain subjects
-- [apply for a student loan](/funding-and-support/tuition-fee-and-maintenance-loans)
+Getting QTS without PGCE may mean that:
 
-You should check the fees for a particular course you are interested in with the course provider.
+* your course is shorter
+* your course fees are lower
+* you submit fewer assignments
 
-## Earn a salary while you train
-These courses are usually only suitable for specific candidates, for example, some are for people who already have relevant teaching experience or are already working as unqualified teachers. You will be able to find out more about eligibility from the training provider. You may be able to study part-time.
+[Find out more about QTS](/what-is-qts).
 
-### School Direct (salaried)
+[Find out more about PGCE](/what-is-a-pgce).
 
-If you have the right experience, School Direct (salaried) give you QTS with many courses also offering a PGCE or PGDE. It takes a year to complete.
+## Non-salaried teacher training
 
-[Find postgraduate training to be a teacher, including School Direct (salaried) courses](https://www.gov.uk/find-postgraduate-teacher-training-courses)
+Most teacher training courses are non-salaried. This means that you have to pay course fees and will not earn a salary while you train.
 
-### Teach First
-Teach First is a 2-year programme leading to a PGDE and QTS. At the end of your first year, you will have QTS. The PGDE is awarded at the end of the second year.
+Fees are usually around £9,250, and can be higher for [international students](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).
+
+If you train to teach certain secondary subjects, you may be eligible for a [scholarship or bursary](/funding-and-support/scholarships-and-bursaries).
+
+No matter what course you choose, you can apply for a student loan.
+
+[Learn about how you could fund your teacher training](/funding-and-support).
+
+### Teacher training providers
+
+Your teacher training course might be provided by a university, school, college or group of schools. Your course might be referred to as:
+
+* school-centred initial teacher training (SCITT)
+* School Direct (there are salaried and non-salaried School Direct courses)
+
+The content of SCITT and School Direct programmes are broadly the same.
+
+Some providers have different course structures and have placements in different kinds of schools, such as special schools. Talk to your provider to find out what they can offer you.
+
+[Find teacher training providers](https://www.find-postgraduate-teacher-training.service.gov.uk/).
```

Comment:
Would we be better talking about courses rather than providers here - Find is 'Find postgraduate teacher training **courses**?

I know you can search by provider but people generally don't (I'll send you the dashboard which shows this)